stellar-multitenant/pkg/serve/serve.go

```diff
@@ -0,0 +1,100 @@
+package serve
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/go-chi/chi/v5"
+	chimiddleware "github.com/go-chi/chi/v5/middleware"
+	supporthttp "github.com/stellar/go/support/http"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/stellar-disbursement-platform-backend/db"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/serve/middleware"
+	"github.com/stellar/stellar-disbursement-platform-backend/stellar-multitenant/pkg/internal/httphandler"
+	"github.com/stellar/stellar-disbursement-platform-backend/stellar-multitenant/pkg/tenant"
+)
+
+type HTTPServerInterface interface {
+	Run(conf supporthttp.Config)
+}
+
+type HTTPServer struct{}
+
+func (h *HTTPServer) Run(conf supporthttp.Config) {
+	supporthttp.Run(conf)
+}
+
+type ServeOptions struct {
+	Environment       string
+	DatabaseDSN       string
+	dbConnectionPool  db.DBConnectionPool
+	GitCommit         string
+	NetworkPassphrase string
```

Comment:
What do we need the NetworkPassphrase for ?

Reply:
This is necessary to know which network type the server is configured to. `Pubnet` or `Testnet`.